applications/The_Grand_Timeline.md

```diff
@@ -0,0 +1,124 @@
+
+## Team
+
+### Team Members
+
+- **Name of Team Leader:** Igor Stepahin
+- **Team Members:** Igor Stepahin (solo project)
+
+### Contact
+
+- **Contact Name:** Igor Stepahin
+- **Contact Email:** igor@grandtimeline.org
+- **Telegram:** [@stepahin](https://t.me/stepahin)
+- **Twitter/X:** [@stepahin](https://x.com/stepahin)
+- **Website:** [grandtimeline.org](https://grandtimeline.org)
+
+### Legal Structure
+
+- **Registered Address:** Gadaunererstrasse 30a, 5630 Bad Hofgastein, Austria
+- **Registered Legal Entity:** Igor Stepahin (no legal entity)
+
+### Team's Experience
+
+Ex. Head of Product Design Gelato Network and Arrakis, 2 years, managed and designed the UI for 4 Gelato products, a design system and a website.
+
+As the sole creator of "The Grand Timeline," I have extensive experience in product design, research, and data visualization. My work on this project has been recognized by notable figures in the crypto community, further validating its value.
+
+### Team Code Repos
+
+Not applicable.
+
+### Team LinkedIn Profiles (if available)
+
+- [Igor Stepahin LinkedIn](https://www.linkedin.com/in/stepahin/)
+
+## "Development" Status
+
+The project has been in development for three years, with significant progress made in researching and documenting the history of crypto and Web3. The current version of the timeline is available on Figma, where I continue to work on it publicly.
+
+![](https://jbm.infura-ipfs.io/ipfs/QmZC3EnHCgVKbQL7JwV9Jk23d92zcbKYkAUYVBTQdPD55j)
+
+Timelapse of the project over the past 3 years — [https://youtu.be/xz2kTPHb2iE](https://youtu.be/xz2kTPHb2iE)
+
+
+**The current status of the project can be called v.0.4 and we can see it in the Figma file where I work publicly: [Figma Link to The Grand Timeline](https://www.figma.com/design/pENhenndHyIMwzW2LgKzNd/The-Grand-Timeline)**
+
+## "Development" Roadmap
+
+### Overview
+
+- **Total Estimated Duration:** 10 months
+- **Full-Time Equivalent (FTE):** 1 FTE
+- **Total Costs:** 60,000 USD
+
+* Website update about the project [grandtimeline.org](http://grandtimeline.org/) // 2 weeks
+* Research and finalization of the Events layer // 8-12 weeks
+* Filling the entire layer of People // 6-8 weeks
+* Static timline version v.1.0 ready for printing // 6-8 weeks
+* UI and prototype of the interactive version // 2-3 weeks
+* Development and launch the interactive version // 4-6 weeks
+
+
+### Milestone 1 — Research and Documentation of Polkadot and Kusama Ecosystems
+
+- **Estimated duration:** 3 months
+- **FTE:** 1
+- **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | Research and documentation of Polkadot and Kusama's history and key developments | A comprehensive section of the timeline detailing key events, milestones, and the impact of Polkadot and Kusama on the broader Web3 ecosystem. |
```

Comment:
Hi @stepahin,
thanks for your application. It would also be great if you could add a License item to your milestone table. See this entry in the [FAQ](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines#license).

Reply:
Hi! Sure, updated.